File: client-unity/Assets/2 - Scripts/entities/PlayerSpawnData.cs
using UnityEngine;

public class PlayerSpawnData
{
	public string playerName;
	public Vector3 position;
	public Vector3 color;

	public PlayerSpawnData(string playerName, Vector3 position, Vector3 color)
	{
		this.playerName = playerName;
		this.position = position;
		this.color = color;
	}

	public override string ToString()
	{
		return $"{playerName},{position.ToString("F4")},{color}";
	}
}

File: server/EzySmashers-app-api/src/main/java/org/youngmonkeys/ezysmashers/app/game/shared/PlayerSpawnData.java
package org.youngmonkeys.ezysmashers.app.game.shared;

import com.tvd12.ezyfox.binding.annotation.EzyObjectBinding;
import com.tvd12.ezyfox.entity.EzyArray;
import lombok.Data;

@Data
@EzyObjectBinding
public class PlayerSpawnData {
	String playerName;
	EzyArray position;
	EzyArray color;
	
	public PlayerSpawnData(String playerName, EzyArray position, EzyArray color) {
		this.playerName = playerName;
		this.position = position;
		this.color = color;
	}
}
